db/seeds.d/90-bookmarks.rb

```diff
@@ -0,0 +1,18 @@
+Bookmark.without_auditing do
+  [
+    { :name => 'failed', :query => "status = failed", :controller => 'job_invocations' },
+    { :name => 'succeeded', :query => 'status = succeeded', :controller => 'job_invocations'},
+    { :name => 'running', :query => 'status = running', :controller => 'job_invocations'},
+    { :name => 'pending', :query => 'status = queued or status = running', :controller => 'job_invocations'},
+    { :name => 'recurring', :query => 'recurring = true', :controller=> 'job_invocations'},
```

Comment:
started_at would cover jobs that already started, right? That's there now, and that's all "recent" would be interested in I think.